lms/djangoapps/commerce/api/v1/models.py

```diff
@@ -88,8 +88,7 @@ def update(self, attrs):
             merged_mode.min_price = posted_mode.min_price
             merged_mode.currency = posted_mode.currency
             merged_mode.sku = posted_mode.sku
-            if posted_mode.expiration_datetime is not None:
```

Comment:
This looks like a very intentional check. I'd like to understand why we felt this was necessary before removing it.

Between this and #11099, it seems like the CAT could be used to create seats with an upgrade deadline but no verification deadline, with no upgrade deadline but a verification deadline, and with neither. The second of those combinations doesn't seem like something we want. Doing so would allow a seat to be purchased after the date by which the purchasing user must have verified in order to receive a certificate.

Does it make sense for a course